src/Microsoft.OData.Core/PropertyMetadataTypeInfo.cs

```diff
@@ -13,7 +14,11 @@ namespace Microsoft.OData
     /// </summary>
     internal class PropertyMetadataTypeInfo
     {
-        public PropertyMetadataTypeInfo(string name, IEdmStructuredType owningType)
+        private bool loadDerivedTypeConstraints;

```

Comment:
>loadDerivedTypeConstraints [](start = 21, length = 26)

nit: sounds like directive (to load or not to load) rather than state. perhaps rename to something like "derivedTypeConstraintsLoaded"

Reply:
#resolved.